source/language/classical.rst

```diff
@@ -110,6 +110,40 @@ multiplication, and division and the corresponding assignment operators.
    angle[10] c;
    c = angle(a + b); // cast to angle[10]
 
+
+Evaluation order
+~~~~~~~~~~~~~~~~
+
+OpenQASM evaluates expressions from left to right.
+The operator precedence is based on C.
+
+   .. table:: [operator-precedence] operator precedence in OpenQASM.
+
+      +-------------------------------+
+      | Operator                      |
+      +-------------------------------+
+      | +x, -x, ~x                    |
```

Comment:
Can you make two columns? One for Operator Type and one for examples. So something like
```
Operator Type: Unary
Example: !, ~, ...
```